lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.ts

```diff
@@ -115,15 +115,6 @@ export class ContentMetadataCardComponent implements OnChanges {
         this.expanded = !this._displayDefaultProperties;
     }
 
-    toggleEdit(): void {
-        this.editable = !this.editable;
-        this.editableChange.emit(this.editable);
```

Comment:
editableChange is defined but unused, please resolve that problem

Reply:
@Yasa-Nataliya this comment is not addressed, please have a look

Reply:
@AleksanderSklorz, Please check this PR:ACA https://github.com/Alfresco/alfresco-content-app/pull/3414 and run this ACA PR with ADF(ACS-5645).